entities/capitalist_card.rb

```diff
@@ -0,0 +1,49 @@
+class CapitalistCard < BaseCard
+  def initialize
+    super
+    @balance = DEFAULT_BALANCE_CAPITALIST
+    @type = CARD_TYPES[1]
+  end
+
+  def number
+    @number ||= generate_card_number
+  end
+
+  def withdraw_tax(amount)
+    tax(amount, withdraw_percent, withdraw_fixed)
+  end
+
+  def put_tax(amount)
+    tax(amount, put_percent, put_fixed)
+  end
+
+  def sender_tax(amount)
+    tax(amount, sender_percent, sender_fixed)
+  end
```

Comment:
Same, this methods are identical for all card implementations, move to BaseCard

Reply:
Done